noetic/distribution.yaml

```diff
@@ -3387,6 +3387,21 @@ repositories:
       url: https://github.com/ros4hri/hri_msgs.git
       version: master
     status: developed
+  hri_rviz:
+    doc:
+      type: git
+      url: https://github.com/ros4hri/hri_rviz.git
+      version: main
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/ros4hri/hri_rviz-release.git
+      version: 0.3.1-1
+    source:
+      type: git
+      url: https://github.com/ros4hri/hri_rviz.git
```

Comment:
Please add a LICENSE file to the source repository (see our [review guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md#types-of-pull-requests)).

Reply:
done.

Reply:
Thank you!